marshmallow/marshalling.py

```diff
@@ -254,11 +241,61 @@ def deserialize(
                         set_value(ret, key, value)
                     elif unknown == RAISE:
                         self.store_error(
+                            ['Unknown field.'],
                             key,
-                            ('Unknown field.',),
                             (index if index_errors else None),
                         )
         return ret
 
     # Make an instance callable
     __call__ = deserialize
+
+
+def merge_errors(errors1, errors2):
+    """Deeply merge two error messages
+
+    Error messages can be string, list of strings or dict of error messages
+    (recursively). Format is the same as accepted by :exc:`ValidationError`.
+    Returns new error messages.
+    """
+    if errors1 is None:
```

Comment:
I think this can be simplified to

```suggestion
    if not errors1:
```

Same with L263.